### PR TITLE
At/cb 200 create the frc modal

### PR DESCRIPTION
**Pull request by Angelbt12**
image

### CODE REVIEW

File: src/components/modals/FrcModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+              <Text appearance="primary" size="large" type="title">
+                Score de riesgo
+              </Text>
+            </Stack>
+
+            <StyledDivider />
+
+            <Stack justifyContent="space-between">
+              <Text appearance="dark" weight="bold">
+                Puntaje total
```
Comment (Brandonz07) [inline src/components/modals/FrcModal/index.tsx:90]: add type and size to text

```<Text appearance="dark" weight="bold" type="label" size="large">```

Do this with the other texts

File: src/components/modals/FrcModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+          </Stack>
+
+          <StyledDivider />
+
+          <Textfield
+            value={maxIndebtedness}
+            iconBefore={<MdOutlineAttachMoney />}
+            fullwidth={true}
+            id="id"
+            label="Endeudamiento máximo"
```
Comment (Brandonz07) [inline src/components/modals/FrcModal/index.tsx:174]: add the type="number"

File: src/components/modals/FrcModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    throw new Error("Portal node not found.");
+  }
+
+  const isMobile = useMediaQuery("(max-width: 700px)");
+
+  return createPortal(
+    <Blanket>
+      <StyledModal $smallScreen={isMobile}>
+        <Stack justifyContent="space-between" alignItems="center">
+          <Text type="headline" size="medium" appearance="dark">
```
Comment (Brandonz07) [inline src/components/modals/FrcModal/index.tsx:54]: size="small"

File: src/components/modals/FrcModal/index.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/components/modals/FrcModal/index.tsx:90]: ok

**Review by Brandonz07 — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

File: src/components/modals/FrcModal/FrcConfig.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export const frcConfig = {
+    title: "Endeudamiento máximo x FRC",
+    subTitle: "Score de riesgo",
+    puntajeTotalLabel: "Puntaje total",
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/modals/FrcModal/FrcConfig.ts]: ok

File: src/components/modals/FrcModal/index.tsx [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+import { frcConfig } from "./FrcConfig";
+import { StyledContainerClose, StyledModal } from "./styles";
+
+export interface ScoreModalProps {
+  title: string;
+  handleClose: () => void;
+  subTitle: string;
+  portalId?: string;
+  puntajeTotal: number;
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/modals/FrcModal/index.tsx]: ok

**Pull request merged**